kobo/client/__init__.py

```diff
@@ -93,13 +93,20 @@ def run(self, *args, **kwargs):
 )
 
 
-class ClientCommandContainer(kobo.cli.CommandContainer):
-
-    def __init__(self, conf, **kwargs):
+class BaseClientCommandContainer(kobo.cli.CommandContainer):
+    def __init__(self):
         self.conf = kobo.conf.PyConfigParser()
+
+    def load(self, conf, **kwargs):
```

Comment:
We should avoid adding a `load` method due to backwards-compatibility concerns.

This class is designed to be subclassed.  Unfortunately, adding new methods to any library class which is intended to be subclassed is a risky matter as you have no idea if someone using the library happens to have also implemented a method named "load" on their subclass.  If they have, then your call to `self.load` later calls *their* method and not this one, and that'll most likely crash.

This is only called from one place anyway, could we just have the self.conf.load_from* moved to ClientCommandContainer constructor?